pkg/apis/crd/v1alpha1/types.go

```diff
@@ -533,6 +537,12 @@ const (
 	NamespaceMatchSelf NamespaceMatchType = "Self"
 )
 
+type PeerScope string
+
+const (
+	ScopeClusterSet PeerScope = "ClusterSet"
```

Comment:
I think we should define a ScopeCluster as well.

Reply:
Done